shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -526,7 +526,7 @@ - (void)loadView {
   // This is an arbitrary offset that is not CGPointZero.
   scrollView.contentOffset = CGPointMake(kScrollViewContentSize, kScrollViewContentSize);
 
-  [self.view addSubview:self.scrollView];
+  [self.view addSubview:scrollView];
```

Comment:
And that's really the most important part of creating the scrollView – the adding it as a subview.

![image](https://github.com/user-attachments/assets/117f6236-e231-4f06-a4f3-d03db5576caf)